src/Queue.php

```diff
@@ -43,14 +43,17 @@ public function push(MessageInterface $message): void
     {
         $this->checkAdapter();
 
-        $this->logger->debug('Preparing to push message "{message}".', ['message' => $message->getName()]);
+        $this->logger->debug(
+            'Preparing to push message with handler name "{handlerName}".',
+            ['handlerName' => $message->getHandlerName()]
+        );
 
         /** @psalm-suppress PossiblyNullReference */
         $this->adapter->push($message);
 
         $this->logger->debug(
-            'Successfully pushed message "{name}" to the queue.',
-            ['name' => $message->getName()]
+            'Successfully pushed message with handler name "{handlerName}" to the queue. Assigned ID #{id}.',
+            ['name' => $message->getHandlerName(), 'id' => $message->getId()]
```

Comment:
Hi! Id can be `null` here, so for this situation to be clear in the log message I suggest to do like this:
```suggestion
            ['name' => $message->getHandlerName(), 'id' => $message->getId() ?? 'null']
```

Reply:
MB. Fixed.

Reply:
Should I press "Resolve conversation"?